### PR TITLE
Rename early access configs

### PR DESCRIPTION
**Pull request by JenySadadia**
Related to https://github.com/kernelci/kernelci-core/pull/2638
- Rename configurations for `early-access` to `production`
- Remove `lava-collabora-early-access` lab

### CODE REVIEW

File: config/pipeline.yaml [hunk 1]
```diff
@@ -130,10 +130,10 @@ storage:
     port: 8022
     base_url: http://172.17.0.1:8002/
 
-  early-access-azure: &azure-files
+  production-azure: &azure-files
     storage_type: azure
     base_url: https://kciapistagingstorage1.file.core.windows.net/
-    share: early-access
+    share: production
```
Comment (JenySadadia) [inline config/pipeline.yaml:136]: @nuclearcat Could you please confirm if updating `share` value won't break storage?

File: config/pipeline.yaml [hunk 2]
```diff
@@ -212,14 +212,6 @@ runtimes:
...
         token: kernelci-api-token-staging
 
-  # ToDo: avoid creating a separate Runtime entry
-  # https://github.com/kernelci/kernelci-core/issues/2088
-  lava-collabora-early-access:
-    <<: *lava-collabora-staging
-    notify:
-      callback:
-        token: kernelci-api-token-early-access
-
```
Comment (JenySadadia) [inline config/pipeline.yaml:222]: I couldn't find any job using this lab.
Is it safe to drop this lab? @nuclearcat

Comment (nuclearcat) [reply config/pipeline.yaml:222]: seems yes, thanks for cleanup!

**Comment by nuclearcat** [comment]
There is at least one file conflicting, can you fix pipeline.yaml so we can run over staging?

**Comment by JenySadadia** [comment]
> There is at least one file conflicting, can you fix pipeline.yaml so we can run over staging?

Resolved the conflict and deployed on staging along with https://github.com/kernelci/kernelci-core/pull/2638.

**Review by pawiecz — APPROVED** [review]
LGTM

The same deployment policy I mentioned in [kernelci/kernelci-core#2638](https://github.com/kernelci/kernelci-core/pull/2638#pullrequestreview-2444859623) applies here

**Pull request merged**